content/courses/testing.md

```diff
@@ -6,19 +6,22 @@ slug: "/courses/Testing/"
 category: "Automated Testing"
 tags:
   - "Jest"
-description: "Welcome to the Automated Testing course. Over the upcoming weeks you will dive into 
-frontend testing using Jest and eventually Enzyme. By the end of this track you will know how 
-to test user interfaces, as well as user interactions." 
-lessons: 
+description: "Welcome to the Automated Testing course. Over the upcoming weeks you will dive into
+frontend testing using Jest, and eventually Enzyme or React Testing Library. By the end of this track you will know how
```

Comment:
While we're here, can we improve the grammar? `Front-End Testing` on this line, and then `By the end of this track___ , ___` (add a comma please).